tests/ParserExceptionsTest.php

```diff
@@ -205,7 +205,7 @@ public function testWrongTagPlacement(): void
         GHERKIN;
 
         $this->expectExceptionObject(
-            new ParserException('Expected Scenario, Outline or Background, but got Step on line: 6')
+            new ParserException('Step can not be tagged, but it is on line: 6')
```

Comment:
I think this new exception is clearer than the previous anyway. I have left the assertion as `ParserException` (rather than change to the actual `UnexpectedTaggedNodeException`) to prove that this is all still BC.